workflow/python/standard-pack/v1/README.md

```diff
@@ -21,26 +21,31 @@ jq             1.6
 unzip          6.00
 ```
 
-### Python Libraries Installed
+### Python libraries installed
 ```
 boto3                 1.22.7
 psycopg2              2.9.3
 snowflake-sqlalchemy  1.3.2
 sqlalchemy            1.4.36
 ```
 
-### Build Arguments
-The Dockerfile expects the following build arguments:
-- `PEAK_USER_ID`: This is the default user that the workflow step runs with when this image is used in the Peak platform. On the Peak platform, this value must be `8877`. Inside the image we create a new user with this user id which is then used when running the image across various services in the Peak Platform.
+### Build arguments
+The Dockerfile expects the following build argument:
+
+`PEAK_USER_ID`
+
+On the Peak platform, its value must be `8877`. 
```

Comment:
Can we skip the value here? in case it changes sometime in future?
Also we can status here that we provide it as a default build arg, so if you need to use it in docker file, how can they use it, a link to an example can be incluced here?

Reply:
it will also help notifying them that in case of non root user they might need to look into giving certain permissions in dockerfiles. so the example link would help here

Reply:
Removed the value and have added a link to the `non-root-user.md` file which has all the details about `PEAK_USER_ID`.